components/02-molecules/text-with-image/_text-with-image.scss

```diff
@@ -8,12 +8,12 @@
   display: grid;
 
   @media (max-width: tokens.$break-mobile-max) {
-    gap: var(--size-spacing-5);
+    gap: var(--size-spacing-6);
   }
 
   @media (min-width: tokens.$break-mobile) {
     grid-template: 'primary secondary';
-    gap: var(--size-spacing-6);
+    gap: var(--size-spacing-8);
```

Comment:
@ahitson I just wanted to point out that we have a couple of "global" gutter sizes:

```
  --spacing-component-gutter: var(--size-spacing-9);
  --spacing-component-gutter-secondary: var(--size-spacing-7);
```

For example, the card grid uses those (`-gutter` for the featured card grid and `-gutter-secondary` for the non-featured grid) and the two-column layout uses `-gutter` for the sidebar.

Did you want to use one of those for this as well? If so, this should change from 8 to 7 or 9. Not sure if making this "line up" with other components is important to you, or if it's desirable to be 8 specifically.

Just wanted to throw that out.

Reply:
@ahitson Like we discussed on zoom. We'll keep this as a custom value for now, and can re-evaluate how components share gutter values later. 👍